src/test/java/TalkingJohn/TJohnTest.java

```diff
@@ -0,0 +1,21 @@
+package TalkingJohn;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TJohnTest {
+
+    @Test
+    public void testEventToString() {
```

Comment:
Can consider using the snake case format to name your tests, so that it is clear from the start what the input and expected output is.